mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/JobClusterActor.java

```diff
@@ -1519,7 +1519,7 @@ private JobDefinition fromJobClusterDefinition(String user, IJobClusterDefinitio
 
 
     private void submitJob(JobDefinition jobDefinition, ActorRef sender, String user) throws PersistException {
-        if(logger.isTraceEnabled()) { logger.trace("Enter submitJobb"); }
+        if (logger.isTraceEnabled()) { logger.trace("Enter submitJobb"); }
```

Comment:
Nit: Get rid of the logger.isTraceEnabled(). The logger message itself is not complex enough to optimize its construction cost.